src/getDataFromMovieDatabaseAPI.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
```

Comment:
I would change the file name (getDataFromMovieDatabaseAPI) to a shorter name because it seems too long and hard to read.

Reply:
Good idea :) Maybe getDataFromMovieDB